### ISSUE TITLE
No interface method isConnected()Z in class Lcom/google/android/gms/common/api/GoogleApiClient

### ISSUE THREAD

**Issue by sabotah** [SeanChristopherConway/titanium-google-geofence#3]
Hi, after creating a custom google-play-services.jar to account for my other plugin includes (for ga.ti and goosh.ti) I seem to get the error above only when compiling this plugin - I have no idea how to solve it

if anyone has the same plugins that I do and have got it working with geofencing, send me your jar file ;)

cheers

**Comment by sabotah** [comment]
I think this has something to do with conflicting versions of google play services to the one I compiled (which doesnt use the isConnected method).

I deleted all my googleservices jars (although I'm pretty sure I will need extra services for goosh and ti.ga) and I'm getting that bindings.json duplication error that you lodged with titanium - should I recompile the src myself with ANT or do you have a working dist that you can commit?

**Comment by SeanChristopherConway** [comment]
I have updated the module and added a working dist that has been compiled with ANT and new module settings and does not contain the cursed bindings.json file. Cheers.

**Comment by SeanChristopherConway** [comment]
Also, @sabotah please refer to this post: https://medium.com/all-titanium/deal-with-titanium-modules-and-its-missing-support-for-android-multidex-546de5486d13#.mpoytsnmp  It is the cleanest way to create a JAR that incorporates all the duplicate classes you will use in your app. This is why Android Studio and Gradle are way better and why Eclipse sucks. There is a reason Google switched 3 years ago. Anyhow, an old headache the Android world solved already that we still have in Appcelerator.

**Comment by sabotah** [comment]
Yup, I used that guide to solve the issue for duplicate classes in the googleservices jar for the modules: ti.ga and ti.goosh - however this error occurs when trying it with this module. I tried using the version you have set in timodule.xml (7571000) as a base for proguard, however classes that I need seem to be missing when attempting to compile the jar for v23 - when compiling for v22 I now get this error when running the app:

Uncaught Error: No virtual method addConnectionCallbacks(Lcom/google/android/gms/common/api/GoogleApiClient$ConnectionCallbacks;)

Could I have a copy of your proguard.txt file? and what googleplayservices version you're using

thanks
